supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchJobDefinition.scala

```diff
@@ -130,45 +123,48 @@ trait AwsBatchJobDefinitionBuilder {
       )
     }
 
-    def buildName(imageName: String, packedCommand: String, volumes: List[Volume], mountPoints: List[MountPoint], env: Seq[KeyValuePair]): String = {
-      val str = s"$imageName:$packedCommand:${volumes.map(_.toString).mkString(",")}:${mountPoints.map(_.toString).mkString(",")}:${env.map(_.toString).mkString(",")}"
 
-      val sha1 = MessageDigest.getInstance("SHA-1")
```

Comment:
You will need to retain the digest and sanitize block so that you end up with a unique name that fits within the allowed size of a job definition. You should also include the retry attempts to distinguish job definitions that do retry from their otherwise identical twins that don't